src/surfaces/filtersDrawer/filtersDrawerMultiSelectLabel.jsx

```diff
@@ -1,7 +1,7 @@
 import _ from 'lodash';
```

Comment:
Should we change "label" to "tag" in this file name and component class name?
`FiltersDrawerMultiSelectLabel` => `FiltersDrawerMultiSelectTag`
`src/surfaces/filtersDrawer/filtersDrawerMultiSelectLabel.jsx` => `src/surfaces/filtersDrawer/filtersDrawerMultiSelectTag.jsx`

If that's more trouble than it's worth, no worries.  Just wondering if we should consider that for consistency of naming ("tag" now instead of "label").

Reply:
Fixed.

Reply:
Looks good.  Resolving.